src/planwise/client/sources/handlers.cljs

```diff
@@ -44,13 +44,14 @@
 (rf/reg-event-db
  :sources.new/created
  in-sources
- (fn [db [_ updated-sources-list]]
+ (fn [db [_ created-source]]
    (rf/dispatch [:modal/hide])
-   (dissoc db :new)
-   (update db :list #(asdf/reset! % updated-sources-list))))
+   (-> db
+       (dissoc :new)
+       (update :list asdf/swap! into [created-source]))))
```

Comment:
You can use `conj` here and avoid creating the single element vector. NB. it's exactly what `into` does https://github.com/clojure/clojure/blob/clojure-1.9.0/src/clj/clojure/core.clj#L6817